libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m

```diff
@@ -87,8 +87,8 @@ - (void) testStoreMultipleEvents {
     NSArray<SFSDKInstrumentationEvent *> *events = [self.storeManager fetchAllEvents];
     XCTAssertTrue(events != nil, @"List of events should not be nil");
     XCTAssertEqual(2, events.count, @"Number of events stored should be 2");
-    XCTAssertTrue([event1 isEqual:[events firstObject]], @"Stored event should be the same as generated event");
```

Comment:
when fetch events from file system, it's not ordered by time stamp, so cannot guarantee the events are as the same sequence as created, thus use [events containsObject: ] to make sure the event exists in the store